README.md

```diff
@@ -64,7 +64,43 @@ The macOS version is a **native Swift port**, developed by [Aryamirsepasi](https
 
 View the [README inside the macOS folder](https://github.com/theJayTea/WritingTools/tree/main/macOS) to learn more.
 
-_(macOS installation instructions coming soon!)_
+#### Installation Instructions:
+
+1. **Install via .dmg file**  
+   - Download the `.dmg` file from the [Releases](https://github.com/theJayTea/WritingTools/releases) page.  
+   - Open the `.dmg` file and drag the `writing-tools.app` into the Applications folder.
+
+2. **Run via building the project**  
```

Comment:
There's a dedicated Build Instructions section toward the bottom, so I'll just move these build instructions there if that's alright :)